akka-http-tests/src/test/scala/akka/http/scaladsl/server/RejectionSpec.scala

```diff
@@ -0,0 +1,21 @@
+package akka.http.scaladsl.server
```

Comment:
I had that for a while, but I didn't figure out how to enable it per-project, so I often had to manually remove it in other sources...